client/src/competition/competition-details-modal/__tests__/competition-details-validation.spec.js

```diff
@@ -0,0 +1,105 @@
+import { validateCompetitionDetails } from '../competition-details-validation';
+
+describe('CompetitionDetailsValidation', () => {
+  describe('validateCompetitionDetails', () => {
+    let sut;
+
+    beforeEach(() => {
+      sut = validateCompetitionDetails;
+    });
+
+    describe('when name changed', () => {
+      it('should be able to save', () => {
+        const competitionDetails = { name: 'test changed', description: 'some description' };
+        const initialCompetitionDetails = { name: 'test', description: 'some description' };
```

Comment:
```initialCompetitionDetails``` variable could be moved to the upper scope, so there will be no need in writing same data in each test

Reply:
Done